workflows/cloud-client/main_test.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import main
+
+
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+
+
+def test_workflow_execution():
+    result = main.execute_workflow(project=PROJECT)
```

Comment:
I think you should specify `location` and `workflow`, `workflow` should use uuid.

Reply:
Can you provide more of a rationale for this?

The location and workflow are default parameters in the quickstart.
The `workflow` parameter is not universally unique (`myFirstWorkflow` is the default) so it can't be a uuid.

If we specify a location and workflow, I'm not sure why we have defaults then.